### DISCUSSION TITLE
Will this project have the same support as vue-virtual-scroller in the nearest future?

### DISCUSSION THREAD

**Discussion by ghost** [Akryum/vue-apollo#1490] [General]
I've found the author of vue-virtual-scroller is developing this project now.

If you will go to https://github.com/Akryum/vue-virtual-scroller, then you will find that the project has a lot of opened bugs and last commit is a half year ago. So it looks like the author doesn't maintain it anymore despite of 8K+ stars he got from users.

So I summon the @Akryum to answer the question. Will this project lose support in the nearest future as well as vue-virtual-scroller? Will @Akryum maintain vue-virtual-scroller or at least mark it to show users it is not maintained anymore **before** they start to use the package?

I don't think the tactics: "create project > get stars > forget about it (and don't let users know it is not maintainable anymore) > start new project" is a good way.

**Comment by ebisbe** [comment]
I'm with you. And that's why now that I'm starting a new project I'm considering switching to https://formidable.com/open-source/urql/docs/basics/vue/
